lib/backend/test/suite.go

```diff
@@ -334,10 +334,10 @@ func (s *BackendSuite) KeepAlive(c *check.C) {
 	c.Assert(string(out.Value), check.Equals, string(item.Value))
 	c.Assert(string(out.Key), check.Equals, string(item.Key))
 
-	err = s.B.KeepAlive(ctx, *lease, addSeconds(time.Now(), 2))
+	err = s.B.KeepAlive(ctx, *lease, addSeconds(time.Now(), 4))
```

Comment:
This seems to minimize the impact of the environment which I believe is a frequent cause of time-related failures as the test processes are subject to system pressure